### ISSUE TITLE
The error caused by the .cache file is incorrect for 2 weeks.

### ISSUE THREAD

**Issue by xingzhefeng** [parcel-bundler/parcel#1695]
首先感谢作者的辛勤付出.
老大,这个错误,大概什么时候能修复?  

Google Translate to English:
```
First of all, thank the author for his hard work.
Boss, this error, when can I fix it?
```

**Comment by DeMoorJasper** [comment]
Would be usefull to have some more information, preferable in English as the translation by Google is quite weird

**Comment by xingzhefeng** [comment]
git pull . 
when I git pull code from my workmate  . parcel watch  or build .  The problem will be reproduced
when I want deploy to server, and git pull and  run parcel  build in server  .  The problem will be reproduced

https://github.com/parcel-bundler/parcel/issues/1656   As like as two peas

**Comment by devongovett** [comment]
dup of #1656. Should be fixed by #1694.
